firmware/gameport-adapter/gameport-adapter.ino

```diff
@@ -54,8 +62,10 @@ static Joystick *createJoystick() {
 }
 
 void setup() {
-  Serial.begin(9600);
-  //while(!Serial);
+  #ifdef __DEBUG__
```

Comment:
I'm not sure about this one, since most of the time, where I'm developing I'm using logging without waiting for the activation of the serial port. The problem with this one is, that the USB initialization doesn't work, if this comes first. So it's a chicken egg problem. I didn't check it out however after I introduced the lambda init function in the loop recently. May this way around we could use both? I guess no, but it's worth the try.

Reply:
It does not seem to make a difference whether `Serial.begin(9600)` is active or not, every joystick was detected successfully either way.